runtime/pkg/util/util.go

```diff
@@ -0,0 +1,12 @@
+// Package util provides general purpose utility functions
+package util
```

Comment:
https://go.dev/blog/package-names#bad-package-names

Reply:
If you want to keep this, `errutil.First(errs ...errors)` would be more idiomatic

Reply:
FYI: there are some third-party "multierror" packages that we use a few places, like https://github.com/hashicorp/go-multierror.

Go has also added it to the std lib in 1.20 (just released, we haven't upgraded yet).

I feel ambivalent about multierrors (often, just returning the first is fine), but sometimes they have a place.

Reply:
I removed the package for now.